.editorconfig

```diff
@@ -14,10 +14,12 @@ spelling_exclusion_path = SpellingExclusions.dic
 indent_size = 4
 insert_final_newline = true
 charset = utf-8-bom
+end_of_line = lf
 
 # XML project files
 [*.{csproj,vbproj,vcxproj,vcxproj.filters,proj,projitems,shproj}]
-indent_size = 4
+indent_size = 2
```

Comment:
current files were using a size of 2 and lf line endings, so I changed it (because my VS defaults are different)